src/DataFixtures/AdminFixtures.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Admin;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class AdminFixtures extends Fixture
```

Comment:
You should segment fixtures by env, not by feature. Something like `DevFixtures` would be easier to work with: references would be kept in the class.